portfolio/src/main/webapp/script.js

```diff
@@ -36,6 +36,36 @@ function randQuote() {
   }
 }
 
+function appendElement(currentElement, newElement) {
```

Comment:
I like how you've structured the DOM manipulation here into clearly named functions!